cmd/algokey/multisig.go

```diff
@@ -50,11 +50,11 @@ func init() {
 	multisigCmd.Flags().StringVarP(&multisigOutfile, "outfile", "o", "", "Transaction output filename")
 	multisigCmd.MarkFlagRequired("outfile")
 
-	convertCmd.Flags().StringVarP(&msigParams, "params", "p", "", "Multisig params -  Threshold PK1 PK2 ...")
-	convertCmd.MarkFlagRequired("params")
-	convertCmd.Flags().StringVarP(&multisigTxfile, "txfile", "t", "", "Transaction input filename")
-	convertCmd.MarkFlagRequired("txfile")
-	convertCmd.Flags().StringVarP(&multisigOutfile, "outfile", "o", "", "Transaction output filename. If not specified, the original file will be modified")
+	appendAuthAddrCmd.Flags().StringVarP(&msigParams, "params", "p", "", "Multisig pre image parameters - [threshold] [Address 1] [Address 2]")
```

Comment:
I'd like to see the help be more explicit on the address ordering:

	appendAuthAddrCmd.Flags().StringVarP(&msigParams, "params", "p", "", "Multisig pre image parameters - [threshold] [Address 1] [Address 2] .. [Address n]")

Reply:
@ryanRfox,

We need to converge the way we present multiple optional parameters. For other UXs, we've used the following:
"inspect [input file 1] [input file 2]..."
"new [address 1] [address 2]..."
"merge -o [merged transaction file] [input file 1] [input file 2]..."

I don't mind picking any particular "pattern", but would like to ensure they are all the same.

Reply:
I agree, we must use a consistent pattern. In this case, please use what is established elsewhere and disregard my comment on this PR.